packages/design-picker/src/utils/global-styles.ts

```diff
@@ -0,0 +1,5 @@
+import { DEFAULT_GLOBAL_STYLES_VARIATION_SLUG } from '../constants';
+
+export function isDefaultGlobalStylesVariationSlug( slug?: string ): boolean {
```

Comment:
Maybe we can move this into `@automattic/global-styles` package as it also has the [isDefaultGlobalStyleVariationSlug](https://github.com/Automattic/wp-calypso/blob/2bb04b6f040a6a2464fd1d17942b150be1b34b71/packages/global-styles/src/components/global-styles-variations/index.tsx#L31) function

Reply:
I thought about it, but `@automattic/global-styles` should be loaded async. I'm not sure if importing it from there would add a lot to the entire JS bundle. WDYT @arthur791004

Reply:
I guess the tree shaking can handle this 🤔 Let's make a minor change to test it!